python/cugraph-pyg/cugraph_pyg/loader/cugraph_node_loader.py

```diff
@@ -154,14 +154,23 @@ def __init__(
         if isinstance(num_neighbors, dict):
             raise ValueError("num_neighbors dict is currently unsupported!")
 
+        renumber = (
+            True
+            if (
+                (len(self.__graph_store.node_types) == 1)
+                and (len(self.__graph_store.edge_types) == 1)
+            )
+            else False
+        )
+
         bulk_sampler = BulkSampler(
             batch_size,
             self.__directory.name,
             self.__graph_store._subgraph(edge_types),
             fanout_vals=num_neighbors,
             with_replacement=replace,
             batches_per_partition=self.__batches_per_partition,
-            **kwargs,
+            renumber=renumber**kwargs,
```

Comment:
Is this valid syntax that I'm not familiar with, or is there a missing comma?

Reply:
definitely a typo, just fixed it

Reply:
ok thanks, I just approved it.  I wasn't paying attention to where CI was when I reviewed it, but if the tests had passed successfully then I think we'd have some missing coverage.  Do you happen to know if the tests cover that?

Reply:
I think you are right about coverage, I'm going to add a test

Reply:
actually, no, it is covered

Reply:
Yes, it failed.  So we do have coverage.